File: selectionSort.js
let arr = [64,25,11,22,12]
let n = arr.length
let min_index
let first_index 

for(i = 0 ; i< n-1 ; i++){
    first_index = i
    let min_value = arr[i]
    for(j = i+1; j< n; j++){
        if(arr[j]< min_value){
            min_value = arr[j]
            min_index = j
        }
    }
    let temp = arr[min_index]
    arr[min_index] = arr[first_index]
    arr[first_index] = temp
}
console.log(arr)

File: Selection_sort.py
arr = [64,25,11,22,12]
n = len(arr)
min_value = arr[0]

for i in range(len(arr) -1):
    first_index = i
    min_value = arr[i]
    for j in range(i+1,len(arr)):
        if(arr[j]< min_value):
            min_value = arr[j]
            min_index= j
    arr[min_index],arr[first_index]  = arr[first_index],arr[min_index]
    print(arr)